katsdpdatawriter/rechunk.py

```diff
@@ -151,11 +152,12 @@ class _Item:
         """
         def __init__(self, offset: Offset, initial_value: np.ndarray, present: bool) -> None:
             self.offset = offset
-            self.value = initial_value
+            self.value: Optional[np.ndarray] = initial_value
```

Comment:
Why `Optional` and not just `np.ndarray`?

Reply:
Exactly. Without the annotation it inferred just ndarray, and then objected to assigning None.